pyuvdata/uvdata/tests/test_miriad.py

```diff
@@ -294,6 +294,8 @@ def test_wronglatlon():
 
 
 def test_miriad_location_handling(tmp_path):
+    from pyuvdata.uvdata import aipy_extracts
```

Comment:
do we want to just make this an `importorskip` too?

Reply:
Good point, I think I have an `importorskip("pyuvdata._miriad")` at the top, but I can switch to `pyuvdata.aipy_extracts`.

Reply:
I was just thinking having both honestly

Reply:
but the aipy_extracts would also include the `_miriad` wouldn't it?

Reply:
Yeah I could do it either way, whichever you think is best. Can't hurt to be doubly sure, so I can change these to be `importorskip`.

Reply:
My first naive thought was just two `importorskips` at the top. I would say just test it out with the single one for aipy extracts and if that works we can go with that.

Reply:
Based on [the example in the pytest documentation](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency), I've now assigned the result of the `importorskip` statements to the module we then import from.

Reply:
Nice looks much cleaner this way.